elements/rh-accordion/demo/color-context.html

```diff
@@ -42,6 +42,13 @@ <h3>Item Three</h3>
   </rh-accordion>
 </rh-context-demo>
 
+<style>
```

Comment:
this doesn't seem to work 


Maybe this change to the `rh-context-demo`
```
#provider {
    padding: var(--rh-context-demo-padding, var(--rh-space-xl, 24px));
    height: auto;
    min-height: 100%;
}
```

Reply:
That will fix it on next release

In prod, playgrounds load packages from unpkg